grammars/javascript.cson

```diff
@@ -617,16 +617,30 @@
     'name': 'meta.class.instance.constructor'
   }
   {
-    'match': '\\b(console)\\b'
-    'name': 'entity.name.type.object.js.console'
-  }
-  {
-    'match': '\\b(\\.)(assert|clear|debug|error|info|log|profile|profileEnd|time|timeEnd|warn)(?=\\()'
-    'captures':
-      '1':
-        'name': 'meta.delimiter.method.period.js'
-      '2':
-        'name': 'support.function.js.console'
+    # console
+    # console.log(arg1, "arg2", [...])
+    'begin': '\\bconsole\\b'
+    'beginCaptures':
+      '0':
+        'name': 'entity.name.type.object.console.js'
+    'end': '(?!\\G)'
+    'patterns': [
+      {
+        'begin': '\\b(\\.)(assert|clear|debug|error|info|log|profile|profileEnd|time|timeEnd|warn)\\s*(?=\\()'
```

Comment:
`\\s*(\\.)\\s*`
Have to test for spaces here as well.  I don't think we need the `\\b` anymore if we test for spaces.